augur/data/lat_longs.tsv

```diff
@@ -133,25 +138,30 @@ country	Maldives	4.7064352	73.3287853
 country	Mali	16.3700359	-2.2900239
 country	Malta	35.8885993	14.4476911
 country	Marshall Islands	6.9518742	170.9985095
+country	Martinique	14.6367927	-61.01582685063731
 country	Mauritania	20.2540382	-9.2399263
 country	Mauritius	-20.2759451	57.5703566
+country	Mayotte	-12.8253862	45.148626111147614
 country	Mexico	19.4325301	-99.1332101
 country	Micronesia	7.0	155.25
 country	Moldova	47.2879608	28.5670941
 country	Monaco	43.73844905	7.424224092532953
 country	Mongolia	47.504782	102.997672
 country	Montenegro	42.9868853	19.5180992
+country	Montserrat	16.73603205085212	-62.19174679903016
 country	Morocco	32.549785	-5.480765
 country	Mozambique	-19.302233	34.9144977
 country	Myanmar	20.600856	96.287616
 country	Namibia	-23.2335499	17.3231107
 country	Nepal	28.0	84.0
 country	Netherlands	52.1326	5.2913
+country	New Caledonia	-21.3019905	165.4880773
 country	New Zealand	-41.500083	172.8344077
 country	Nicaragua	12.6090157	-85.2936911
 country	Niger	18.061703	10.145544
 country	Nigeria	10.0	8.0
 country	North Macedonia	41.6171214	21.7168387
+country	Northern Mariana Islands	14.149020499999999	145.21345248318923
```

Comment:
is there any value to trying to truncate some of these to a standard number of significant digits? Picking this one particularly because that latitude looks a whole lot like a floating-point representation issue.

Reply:
The lat/longs are included in the exported Auspice JSON if the dataset has a matching location. So if we truncate these, it could potentially decrease the size of the Auspice JSONs. 

Let's say, on average we truncate by 5 digits per value, so that's 10 bytes per location. If a dataset included _all_ of the locations of this lat_longs.tsv, the max reduction is 10 * 3213 = 32,130 bytes or 0.03213 MB. 
Probs not worth it 🤷‍♀️